From 110051631d933277beaf2b1a88235d464669a84c Mon Sep 17 00:00:00 2001
From: Wendy Liang <wendy.liang@xilinx.com>
Date: Mon, 10 Aug 2020 22:21:05 -0700
Subject: [PATCH 1508/1851] misc: xilinx-ai-engine: replace u32 with __u32 in
 ioctl struct

commit ed447247557780d63f9daff67e859161d29a5abe from
https://github.com/Xilinx/linux-xlnx.git

Replace u32 with __u32 in the AI engine DMA struct which will be used in
SHIM DMA buffer descriptor setting ioctl.

Signed-off-by: Wendy Liang <wendy.liang@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 include/uapi/linux/xlnx-ai-engine.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/uapi/linux/xlnx-ai-engine.h b/include/uapi/linux/xlnx-ai-engine.h
index 6616fe4503bb..579f31c4695a 100644
--- a/include/uapi/linux/xlnx-ai-engine.h
+++ b/include/uapi/linux/xlnx-ai-engine.h
@@ -135,7 +135,7 @@ struct aie_partition_req {
  * @bd_id: buffer descriptor id
  */
 struct aie_dma_bd_args {
-	u32 *bd;
+	__u32 *bd;
 	__u64 data_va;
 	struct aie_location loc;
 	__u32 bd_id;
-- 
2.31.1

